devito/ir/support/basic.py

```diff
@@ -2,6 +2,7 @@
 from functools import cached_property
 
 from sympy import S
+import sympy
```

Comment:
jusit import Interval above no?

Reply:
I would, but to make a neat distinction with our own Interval...